integration_tests/suite/test_config.py

```diff
@@ -19,6 +20,7 @@ class TestConfig(BaseIntegrationTest):
     wait_strategy = NoWaitStrategy()
 
     def test_config(self):
+        sleep(5)  # Necessary wait for services to initialize
```

Comment:
This depend of the test execution order
Should be in setUpClass to remove this dependency
Also the comment should be explicit about which step of the initialization process we are waiting for